src/makeCalculator.js

```diff
@@ -4,7 +4,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add(num) {
+      this.result += num;
+    },
+    subtract(num) {
+      this.result -= num;
+    },
+    multiply(num) {
+      this.result *= num;
+    },
+    divide(num) {
+      if (num !== 0) {
+        this.result /= num;
+      } else {
+        this.result = null;
```

Comment:
[CODE STYLE] - Instead of assigning `null` to `this.result` when division by zero occurs, it's more conventional to handle the error differently. Setting the result to `null` might be unexpected for users of the calculator. Consider throwing an error, or setting the result to `Infinity`, which is the result of division by zero in JavaScript.